src/bun.js/bindings/ProcessBindingConstants.cpp

```diff
@@ -905,20 +905,20 @@ static JSValue processBindingConstantsGetCrypto(VM& vm, JSObject* bindingObject)
 #ifdef RSA_PKCS1_OAEP_PADDING
     object->putDirect(vm, PropertyName(Identifier::fromString(vm, "RSA_PKCS1_OAEP_PADDING"_s)), jsNumber(RSA_PKCS1_OAEP_PADDING));
 #endif
-#ifdef RSA_X931_PADDING
-    object->putDirect(vm, PropertyName(Identifier::fromString(vm, "RSA_X931_PADDING"_s)), jsNumber(RSA_X931_PADDING));
+#ifndef RSA_X931_PADDING
+    object->putDirect(vm, PropertyName(Identifier::fromString(vm, "RSA_X931_PADDING"_s)), jsNumber(5));
 #endif
```

Comment:
Thank you @saklani! We still want to use the values if they exist so we shouldn't remove the `#ifdef` but we should add an `#else` and put the default value.

Reply:
Done!